### PR TITLE
push

### PR DESCRIPTION
**Pull request by Shadow090509**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,28 +12,30 @@ import { Routes, Route } from "react-router-dom";
...
+    //   headers: {
+    //     "Content-Type": "application/json"
+    //   },
+    //   // HTTP verb so the correct endpoint is invoked on the server
+    //   method: "POST"
+    // })
+    //   .then((response) => response.json())
+    //   .then((payload) => readCat())
+    //   .catch((errors) => console.log("Cat create errors:", errors))
   };
```
Comment (WalkerTexas-coder) [inline src/App.js]: Lets remove this code for create cat fetch from the read functionality branch

File: src/pages/CatShow.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+describe("<CatShow />", () => {
+  it("renders a card for each cat", () => {
+    render(<CatShow cats={mockCats} />);
+
+    mockCats.map((cat) => {
+      const nameElement = screen.getByText(cat.name);
+      expect(nameElement).toBeInTheDocument();
+    });
+  });
```
Comment (WalkerTexas-coder) [inline src/pages/CatShow.test.js:24]: Let's add some tests for the showCat page using memoryRouter.

**Review by WalkerTexas-coder — COMMENTED** [review]
Couple of comments to address

File: src/pages/CatNew.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const CatNew = ({ createCat }) => {
     enjoys: "",
     image: "",
   });
-
+  console.log(newCat);
```
Comment (elysemontano) [inline src/pages/CatNew.js]: Let's remove this console log

**Review by elysemontano — APPROVED** [review]
Please remove console log on CatNew before merging

**Pull request merged**